src/app/pages/submission-pages/overview-page/overview-page.component.html

```diff
@@ -34,7 +34,22 @@ <h2 class="padding-top-xlarge">New data submission</h2>
           <input type="radio" *ngIf="!savedControlled" name="controlled" value="0" formControlName="controlled" id="controlledNo" (change)="onUpdateField('controlled')">
             <label for="controlledNo" *ngIf="!savedControlled">No</label>
           <div *ngIf="savedControlled">
-           You have selected : <span class="font-weight-bold">{{savedControlled == 1 ? 'Yes' : 'No'}}</span> <a (click)="onChangeField('controlled')"> (Change)</a>
+           You have selected : <span class="font-weight-bold">{{savedControlled === '1' ? 'Yes' : 'No'}}</span> <a (click)="onChangeField('controlled')"> (Change)</a>
+          </div>
+        </fieldset>
+
+
+        <fieldset class="large-form padding-bottom-large padding-top-large">
+          <legend>This service requires limited processing of your personal data in order to function.<br />By using the site you are agreeing to this as outlined in our <a href="https://www.ebi.ac.uk/data-protection/privacy-notice/ebi-wide-submission" target="_blank">Privacy Notice</a> and <a href="https://www.ebi.ac.uk/about/terms-of-use" target="_blank">Terms of Use</a></legend>
+          <!-- TODO: this should be a checkbox rather than radio button, but need to create styles for it match -->
+          <!-- <input type="checkbox" checked="savedGDPR" name="gdpr" value="1" formControlName="gdpr" id="gdprYes" (change)="onUpdateField('gdpr')">
+            <label for="gdprYes">Yes, I agree</label> -->
+          <input type="radio" *ngIf="!savedGDPR" name="gdpr" value="1" formControlName="gdpr" id="gdprYes" (change)="onUpdateField('gdpr')">
+            <label for="gdprYes" *ngIf="!savedGDPR">Yes, I agree</label>
```

Comment:
Its better if you have label in the same indentation / alignment of input as its not wrapped by input.